x/ethbridge/handler_test.go

```diff
@@ -21,14 +21,17 @@ func TestBasicMsgs(t *testing.T) {
 	valAddress := validatorAddresses[0]
 
 	//Unrecognized type
-	res := handler(ctx, sdk.NewTestMsg())
-	require.False(t, res.IsOK())
-	require.True(t, strings.Contains(res.Log, "unrecognized ethbridge message type: "))
+	res, err := handler(ctx, sdk.NewTestMsg())
+	require.NotNil(t, err)
```

Comment:
suggestion for testing errors
```suggestion
	require.Error(t, err)
```

Reply:
applies for all tests